lib/index.js

```diff
@@ -41,7 +42,25 @@ exports = module.exports = internals.Connection = class {
         }
 
         this._client = new Memcache(this.settings.location, this.settings);
-        this.isConnected = true;
+
+        return new Promise((resolve, reject) => {
+
+            const server = this._client.servers[0];
```

Comment:
The `.servers` property is definitely accessible on the object but it's not documented, at least I couldn't find any mention in their GitHub readme. Should we be using it?